src/api/apiBase.ts

```diff
@@ -35,7 +36,12 @@ const apiBase = (withVersion = true) => {
 export default apiBase;
 
 export function termsBase() {
-  return fixUrl((window as any).ferdium.stores.settings.all.app.server !== LIVE_FRANZ_API
-    ? (window as any).ferdium.stores.settings.all.app.server
-    : DEV_API_FRANZ_WEBSITE);
+
+  const terms = (window as any).ferdium.stores.settings.all.app.server !== LIVE_FRANZ_API
```

Comment:
Can we remove the repetitive code please into a local constant?

Reply:
Instead of a nested ternary operator, a `switch` could also do the trick